src/store/chat/saga.ts

```diff
@@ -82,6 +85,25 @@ export function* setActiveConversation(id: string) {
   history.push({ pathname: `/conversation/${id}` });
 }
 
+export function* performValidateActiveConversation() {
+  yield put(setIsErrorDialogOpen(false));
```

Comment:
Probably don't need this. It should already be closed if we're just now performing the validation.